app/src/main/java/org/oppia/android/app/devoptions/platformparameters/PlatformParametersFragmentPresenter.kt

```diff
@@ -206,55 +296,136 @@ class PlatformParametersFragmentPresenter @Inject constructor(
         if (boundParamIds.contains(id).not()) {
           return@onPlatformParameterTextChangedCallback
         }
+        val originalValue = if (resetParameters.containsKey(model.platformParameterId)) {
```

Comment:
Maybe? since it communicates clearly: the last valid input the system recognizes.
```suggestion
        val lastValidValue = if (resetParameters.containsKey(model.platformParameterId)) {
```

Reply:
updated